src/Utils/CampaignConfigurationLoader.php

```diff
@@ -85,14 +130,16 @@ private function buildBucketFromData( array $bucketData ): Bucket {
 		if ( !isset( $bucketData['name'] ) ) {
 			throw new InvalidConfigurationValueException( 'A configured bucket has no name.' );
 		}
-		if ( !isset( $bucketData['banners'] ) ) {
+
+		$name = $this->getStringValue( $bucketData, 'name' );
```

Comment:
`name` is a very generic term, we could increase readability by calling it `bucketName` for example

Reply:
I have renamed the variable. I hope this is what you meant. If you would like further improvement, please let me know. Thank you.